src/data/polynomial/erase_lead.lean

```diff
@@ -96,6 +96,19 @@ begin
   exact card_lt_card (erase_ssubset $ nat_degree_mem_support_of_nonzero h)
 end
 
+lemma erase_lead_card_support_eq {R : Type*} [semiring R] {c : ℕ} {f : polynomial R}
+  (f0 : f.support.card = c.succ) :
+  f.erase_lead.support.card = c :=
+begin
+  rw [erase_lead_support, card_erase_of_mem],
+  { exact nat.pred_eq_of_eq_succ f0 },
+  { rw nat_degree_eq_support_max',
+    refine f.support.max'_mem (nonempty_support_iff.mpr _),
+    { rintro rfl,
+      rw [support_zero, card_empty] at f0,
+      exact (not_le.mpr (nat.succ_pos c)) (eq.ge f0) } }
```

Comment:
Since there's only one goal after the `refine`, there's no need to introduce another level of nested braces afterwards.
```suggestion
    rintro rfl,
    rw [support_zero, card_empty] at f0,
    exact (not_le.mpr c.succ_pos) f0.ge }
```

Reply:
Thanks for the suggestion!

Reply:
Btw, I had it indented since when I used the refine I had 2 goals, but one of the two disappeared along the way.  I did not know what the standard indentation for this was.